data/pc/latest/proto.yml

```diff
@@ -2120,23 +2120,24 @@
          default: void
 
    # https://github.com/extremeheat/extracted_minecraft_data/blob/client1.21.2/client/net/minecraft/world/entity/Relative.java#L82
-   PositionUpdateRelatives: ["bitflags", {
```

Comment:
What is the reason to replace every bitflags with bitfields? You said it wasn't passing test but do you know why they failed?

Reply:
Oh that's because bitflag is a new type defined [here](https://github.com/PrismarineJS/node-minecraft-protocol/compare/master...1.21.3)